packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -15,7 +15,7 @@
 			>
 				<template #content>
 					<tera-drilldown-section :is-loading="isFetchingPDF">
-						<tera-pdf-panel :pdfs="pdfData" />
+						<tera-pdf-panel :pdfs="pdfData" @pdf-ref="getPdfRef" />
```

Comment:
```suggestion
						<tera-pdf-panel :pdfs="pdfData" ref="pdfPanelRef" />
```
create a `pdfPanelRef` in the js too